flights/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db   import models
+
+from core.models import TimeStampModel
+
+
+class FlightDetail(TimeStampModel): 
+    departure_time  = models.DateTimeField()
+    arrival_time    = models.DateTimeField()
+    remaining_seats = models.IntegerField()
+    reserved_seats  = models.IntegerField()
+    price           = models.DecimalField(max_digits=15, decimal_places=2)
+    flight_route    = models.ForeignKey('FlightRoute', on_delete=models.CASCADE)
+
+    class Meta: 
+        db_table = 'flight_details'
+
+
+class FlightRoute(TimeStampModel): 
+    code        = models.CharField(max_length = 10)
+    departure   = models.ForeignKey('Location', related_name = 'departure_id', on_delete = models.CASCADE)
+    destination = models.ForeignKey('Location', related_name = 'destination_id', on_delete = models.CASCADE)
```

Comment:
related_name을 지정하면 해당 이름으로 역참조를 하게 됩니다.
즉 Location 입장에서 FlightRoute 정보를 찾게 됩니다.
현재 지정하신 이름으로 하게 되면 아래와 같이 코드가 구성되게 됩니다. 
보시다시피 의미가 엉망이 되어 버립니다.

```python
location = Loaction.objects.get(id=1)
depature_flight_routes = location.departure_id.all()
```
아래와 같이 수정 한 후 related_name을 사용할 때 어떤지를 생각해 보면 좋을 것 같아요
```suggestion
    departure   = models.ForeignKey('Location', related_name = 'depature_flight_routes', on_delete = models.CASCADE)
    destination = models.ForeignKey('Location', related_name = 'destination_flight_routes', on_delete = models.CASCADE)
```

Reply:
related_name을 역참조때 flightroute_set대신 쓰기 위해 지정한다는 것을 잊고 있었습니다 컬럼명으로 착각하고 있었네요 감사합니다!😀